src/commands/preboot-simulator.yml

```diff
@@ -4,6 +4,10 @@ parameters:
     description: The device type. E.g., "iPhone 12"
     type: string
     default: "iPhone 12"
+  device-udid-var:
+    description: The name of the variable to contain the device's UDID e.g. IPHONE_UDID
+    type: string
+    default: DEVICE_UDID
```

Comment:
Let's make this something like `MACOS_ORB_DEVICE_UDID` by default. This just makes it explicit as to what the variable is and will prevent any possible collisions with users existing commands/scripts

Reply:
done